src/Composite/CompositeParser.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Db\Pgsql\Composite;
+
+/**
+ * Composite type representation to PHP array parser for PostgreSQL Server.
+ */
+class CompositeParser
+{
+    /**
+     * Converts composite type value from PostgreSQL to PHP array
+     *
+     * @param string $value String to convert.
+     */
+    public function parse(string $value): array|null
```

Comment:
Suggest use static methods and don't create instance of `CompositeParser` in `ColumnSchema` on each value typecasting.

Reply:
I did it according to the design of `ArrayParser`.
Can fix it after the issue #297 will be sollved.